examples/amp-video.amp.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
```

Comment:
I would not merge this file until the final PR. We want all things in examples to be valid.